### DISCUSSION TITLE
v0.17.2

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1237] [Releases]
## What's Changed
* Hotfix v0.17.2 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1236

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.17.1...v0.17.2

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.17.2'>v0.17.2</a>.</em>

**Comment by seh2000** [comment]
Updated from 0.17.0 > 0.17.1 > 0.17.2, not using "latest" but the actual version.
